install/pre-installation.md

```diff
@@ -441,6 +441,17 @@ in `/etc/environment` from the [Download CSM tarball](#21-download-csm-tarball)
            update -y cray-site-init
        ```
 
+   1. Update `iuf-cli`.
+
+       > ***NOTE*** This provides `iuf`, a command line interface to the [Install and Upgrade Framework](../operations/IUF.md).
+
+       ```bash
+       zypper \
+           --plus-repo "${CSM_PATH}/rpm/cray/csm/sle-$(awk -F= '/VERSION=/{gsub(/["-]/, "") ; print tolower($NF)}' /etc/os-release)/" \
+           --no-gpg-checks \
+           update -y iuf-cli
```

Comment:
Will `iuf-cli` be in the LiveCD beforehand? I haven't seen a csm-rpms PR for including it in the LiveCD build.

Wondering if this needs to be changed to `install`, and the step should be "Install `iuf-cli`."

Reply:
I've updated the verbage